clar2wasm/tests/standard/utils.rs

```diff
@@ -13,17 +13,26 @@ pub(crate) fn load_stdlib() -> Result<(Instance, Store<()>), wasmtime::Error> {
     let mut linker = Linker::new(&engine);
 
     // Link in the host interface functions.
+    linker
+        .func_wrap(
+            "clarity",
+            "define_function",
+            |_: Caller<'_, ()>, _kind: i32, _name_offset: i32, _name_length: i32| {
```

Comment:
`caller` shouldn't be needed here (unless you are planning to use it in the function later..)

Reply:
Removed in 0d4047b